torch/_inductor/cpp_builder.py

```diff
@@ -761,6 +764,17 @@ def homebrew_libomp() -> Tuple[bool, str]:
         return False, ""
 
 
+@functools.lru_cache(None)
+def perload_clang_libomp_win(cpp_compiler: str, omp_name: str) -> CDLL:
+    output = subprocess.check_output([cpp_compiler, "-print-file-name=bin"]).decode(
+        "utf8"
+    )
+    omp_path = os.path.join(output.rstrip(), omp_name)
+    os.environ["KMP_DUPLICATE_LIB_OK"] = "TRUE"
```

Comment:
Hi @jgong5 It will get the current c++ compiler's openmp. I write this code if reference Intel compiler team's example code: https://github.com/intel/intel-extension-for-pytorch/commit/d60d6342030e9e85073e1eb731318bb8827d0a26#diff-0c02c9ac6a765541dc01337b42b01fac6758748c2db9007320a8ab9030a59f9aR17-R22

1. `--print-file-name=bin` to get compiler's bin directory.
2. Get the `libomp.dll` in compiler's `bin` directory.
3. Load the compiler's `libomp.dll`.

Reply:
Local test:
<img width="510" alt="image" src="https://github.com/user-attachments/assets/0248da4c-f273-4793-ae03-cec0dbe5765e">